docs/topics/http/urls.txt

```diff
@@ -231,8 +231,11 @@ default argument for ``num``, ``"1"``. If the second pattern matches,
 Performance
 ===========
 
-Each regular expression in a ``urlpatterns`` is compiled the first time it's
-accessed. This makes the system blazingly fast.
+The URL function processes regular expressions in the ``urlpatterns`` list
+which is compiled upon the first http request. Subsequent requests use the
```

Comment:
I think the previous wording "the first time it's accessed" is more accurate since it also accounts for using the URL system in the shell or in tests.